mypy.ini

```diff
@@ -0,0 +1,65 @@
+# Global options:
```

Comment:
Were all of these necessary for the codebase to pass ?

Reply:
No, these are a few default options I like. We can tweak it as we go.

Reply:
OK.